.github/workflows/publish-release.yml

```diff
@@ -4,63 +4,42 @@ on:
   workflow_dispatch:
 
 jobs:
-  build:
+  publish-release:
     runs-on: ubuntu-latest
-
     steps:
       - uses: actions/checkout@v3
-
       - name: Set up JDK 17
         uses: actions/setup-java@v3
         with:
-          distribution: 'temurin'
+          distribution: 'adopt'
```

Comment:
Isn't this `temurin`? `central-publish` workflow is using `temurin`.

Reply:
Fixed.